index.js

```diff
@@ -1,3 +1,19 @@
-const App = require('./App');
+const express = require('express');
+const mongoose = require('mongoose');
+const config = require('./config/db-config');
 
-new App();
\ No newline at end of file
+const startServer = () => {
+  const server = express();
+  server.get('/', (req, res) => res.send('Hello World!'))
+  server.listen(config.PORT, () => console.log(`Listening on port ${config.PORT}`)
+  );
+}
+
+config.URL;
+
+const db = mongoose.connection;
```

Comment:
Move db connection to its own file please

Reply:
dafuq du menar?

Reply:
Din databas-relaterade kod bör ligga i en egen fil, förslagsvis i en config mapp i en db.js fil. I den filen kan du göra din mongoose.connection och dylikt.